src/vm-repair/azext_vm_repair/_help.py

```diff
@@ -35,3 +35,33 @@
           text: >
             az vm repair restore -g MyResourceGroup -n MyVM --disk-name MyDiskCopy --verbose
 """
+
+helps['vm repair run'] = """
+    type: command
+    short-summary: Run verified scripts from GitHub on a VM. 'az vm repair run-list' to view scripts.
+    examples:
+        - name: Run the script with <run-id> on the repair VM.
+          text: >
+            az vm repair run -g MyResourceGroup -n MySourceWinVM --run-id win-hello-world --run-on-repair --verbose
+        - name: Run a script with parameters on the repair VM. Remove --run-on-repair to run directly on VM.
+          text: >
+            az vm repair run -g MyResourceGroup -n MySourceWinVM --run-id win-hello-world --parameters hello=hi world=earth --run-on-repair --verbose
+        - name: Run a custom script on the repair VM.
+          text: >
+            az vm repair run -g MyResourceGroup -n MySourceWinVM --custom-run-file ./file.ps1 --run-on-repair --verbose
+"""
+
+helps['vm repair run-list'] = """
```

Comment:
i think this command could be named differently. "vm repair run-list" might suggest that it list scripts that are currently running or were exexuted on the vm. Perhaps it could be just "vm repair available-scripts"?

Reply:
Yes I think that would be better. I'll make an iteration with the change. Thank you for the suggestion.